docs/src/types/vectors.md

```diff
@@ -12,4 +12,10 @@ You can use a vector just like you would use any other type -- e.g. a `[Vec<u32>
 
 ## Returning vectors
 
-This is currently not supported. If you try returning a type that is or contains a vector you will get a compile time error.
+Returning vectors from contract methods is supported transparently, with the caveat that you cannot have them nested inside another type.
```

Comment:
Maybe hint that this is a temporary limitation, what do you think?

Reply:
Ah yes sure!